templates/components/faceted-search/facets/hierarchy.html

```diff
@@ -3,7 +3,7 @@
     <div
         class="accordion-navigation toggleLink {{#unless ../start_collapsed }} is-open {{/unless}}"
         role="button"
-        data-collapsible="#facetedSearch-content--{{#replace '&' (hyphenate facet) }}{{/replace}}">
+        data-collapsible="#facetedSearch-content--{{#replace '&' (concat '&' (hyphenate facet)) }}{{/replace}}">
```

Comment:
Sorry I am not sure I understand what is going on here and what are we trying to do. Can you please an explanation to the description as to why were they broken in the first place. Also I am not sure I understand why we need to add `concat '&'`. Can you please explain using this example `Custom & Filter`. If I am reading correctly it will look something like this;
`Custom & Filter` -> hyphenate `Custom-&-Filter` -> concat `&Custom-&-Filter` -> replace `Custom-Filter`

Please correct me if I am wrong.

Reply:
replace helper, returns empty string if it doesn't find any `&` in it. Thus all the standard filters where return as `empty string`.  So, we fixed it by `concatenating &` to all the input facets and replace them later. 

Since custom filters are part of this loop, we ended up adding and removing the `&` to it.

Reply:
Open to suggestions here but what happens is if `replace` doesn't find the replacement character, it returns an empty string. So what happens is `Custom Filter` becomes: `'Custom Filter'` -> hyphenate `'Custom-Filter'` -> replace `''`. The `concat` forces there to be at least one replacement token, so it always exists.

Reply:
It feels so hacky and for input's like this `Custom & Filter` it will generate output with double dash`Custom--Filter`.

Reply:
I agree, but the whole way the expand/collapse system works using css selectors also feels just as hacky to me. I'm open to suggestions here, once again.

Reply:
I think it should no longer return empty string based on https://github.com/bigcommerce/paper-handlebars/pull/92/files

Reply:
OK, I figured out what the disconnect was here. We needed to use `{{replace}}` with `{{else}}`, which is beyond confusing. 😕